rsts/user/features/on_failure_policy.rst

```diff
@@ -0,0 +1,45 @@
+.. _on-failuire-policy:
+
+What is it
+==========
+
+The default behavior for when a node fails in a workflow is to immediately abort the entire workflow. The reasoning behind this thinking
+is to avoid wasting resources since the workflow will end up failing anyway. There are certain cases however, when it's desired for the
+workflow to carry on executing the branches it can execute. 
+
+For example when the remaining tasks are marked as :ref:`cacheable <features-task_cache>`. 
+Once the failure has been fixed and the workflow is relaunched, cached tasks will be bypassed quickly.
+
+How to use it
+-------------
+
+Use on_failure attribute on workflow_class.
+
+.. code:: python
+
+   from flytekit.models.core.workflow import WorkflowMetadata
+
+   @workflow_class(on_failure=WorkflowMetadata.OnFailurePolicy.FAIL_AFTER_EXECUTABLE_NODES_COMPLETE)
+   class RunToCompletionWF(object):
+     pass
+
+Available values in the policy:
+
+.. code:: python
+
+    class OnFailurePolicy(object):
```

Comment:
one day we'll figure out how to link to python code... but not this day

Reply:
Agreed... I really wanted to because I know this will go stale and we will not know